packages/fiori-elements-writer/test/lrop.test.ts

```diff
@@ -161,7 +163,7 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
                         settings: v2TemplateSettings
                     },
                     ui5: {
-                        version: '1.77.2' // flex changes preview should be included with this version
+                        version: '1.108.0'
```

Comment:
Why was this changed?

Reply:
Because the FE V2 types are only available in `~1.108.0` and `>=1.110.0`.

Reply:
But it appears that the version chosen is to test the switching on of a feature (`flex changes`) not TS types. But maybe this is already covered in a non-TS test?

Reply:
Yeah that's why I removed the flex changes comment as it is covered by the JS version.